database/seeds/ReportbackItemSeeder.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Rogue\Models\Reportback;
+use Rogue\Models\ReportbackItem;
+use Illuminate\Database\Seeder;
+
+class ReportbackItemSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $faker = Faker\Factory::create();
+
+        $reportbacks = Reportback::all();
```

Comment:
Not necessary for merge, but it could make sense to do this with model factories in the `ReportbackSeeder` seeder. Check out the "Adding Relations To Models" section in the [Model Factories](https://laravel.com/docs/5.2/testing#model-factories) docs.